lib/data-types.js

```diff
@@ -388,6 +455,9 @@ var JSONTYPE = function() {
 util.inherits(JSONTYPE, ABSTRACT);
 
 JSONTYPE.prototype.key = JSONTYPE.key = 'JSON';
+JSONTYPE.prototype.validate = function(value) {
+  return true;
```

Comment:
Can't really validate here I think - json objects, plain strings, booleans and nulls are all allowed